3-0-libraries.md

```diff
@@ -37,7 +37,7 @@ Download the worksheet for this section here: <a href="3-0-libraries-worksheet">
 A _library_ is made from one sample for a single target _platform_ and
 has a specific _design_ associated with it that decides the _selection_
 and _strategies_ used to make the library. A library may also have _indices_
-(primers) and QC information.
+(primers/barcodes/molecular IDs) and QC information.
```

Comment:
I think it's worth keeping something like "barcodes" in for training new users, who may be coming from somewhere where they call them barcodes. Perhaps "sequencing barcodes"?